src/Lachain.Consensus/AbstractProtocol.cs

```diff
@@ -92,7 +92,7 @@ public void Terminate()
             lock (_queueLock)
             {
                 if (Terminated) return;
-                Logger.LogTrace($"{Id}: protocol is terminated");
+            //    Logger.LogTrace($"{Id}: protocol is terminated");
```

Comment:
4.5%

Reply:
Better to have this record,  important for consensus state understanding